### PR TITLE
add s3 known limitations to sigv4

### PR DESCRIPTION
**Pull request by HugoPct**
### Motivation
Add known limitations.
### Preview

https://docs-staging.datadoghq.com/hugo.puceat/sigv4-2/synthetics/multistep

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/api_tests/http_tests.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ After choosing to create an [`HTTP` test][4], define your test's request.
 
    * **HTTP Basic Auth**: Add HTTP basic authentication credentials.
    * **AWS Signature V4**: Enter your Access Key ID and Secret Access Key. Datadog generates the signature for your request.
+   This option uses the basic implementation of SigV4.
+   Known limitations: AWS S3 specific signature is not implemented.
```
Comment (alai97) [inline content/en/synthetics/api_tests/http_tests.md:54]:
```suggestion
   This option uses the basic implementation of SigV4. AWS S3-specific signatures are not implemented.
```

File: content/en/synthetics/multistep.md [hunk 2]
```diff
@@ -74,6 +74,8 @@ To create an HTTP request step, click **Create Your First Step**.
 
    * **HTTP Basic Auth**: Add HTTP basic authentication credentials.
    * **AWS Signature V4**: Enter your Access Key ID and Secret Access Key. Datadog generates the signature for your request.
+   This option uses the basic implementation of SigV4.
+   Known limitations: AWS S3 specific signature is not implemented.
```
Comment (alai97) [inline content/en/synthetics/multistep.md:78]:
```suggestion
   This option uses the basic implementation of SigV4. AWS S3-specific signatures are not implemented.
```

File: content/en/synthetics/api_tests/http_tests.md (hunk 1 above)
Comment (cswatt) [inline content/en/synthetics/api_tests/http_tests.md:54]:
```suggestion
   This option uses the basic implementation of SigV4. The AWS S3 specific signature is not implemented.
```

**Review by alai97 — APPROVED** [review]
Looks good pending a small copy nit!

File: content/en/synthetics/multistep.md (hunk 2 above)
Comment (cswatt) [inline content/en/synthetics/multistep.md:78]:
```suggestion
   This option uses the basic implementation of SigV4. The AWS S3 specific signature is not implemented.
```

**Review by cswatt — APPROVED** [review]

**Pull request merged**